include/highfive/bits/H5Converter_misc.hpp

```diff
@@ -57,14 +60,22 @@ struct DeepCopyBuffer {
         : buffer(inspector<T>::getSize(_dims))
         , dims(_dims) {}
 
-    hdf5_type* get_pointer() {
+    hdf5_type* getPointer() {
         return buffer.data();
     }
 
-    hdf5_type const* get_pointer() const {
+    hdf5_type const* getPointer() const {
         return buffer.data();
     }
 
+    hdf5_type* begin() {
+        return getPointer();
+    }
+
+    hdf5_type const* begin() const {
```

Comment:
Here we get a `const* begin`, but not above? What about `cbegin`?

Reply:
It's the same as with say `std::vector`, if the object is const, then so is the iterator. See,
https://en.cppreference.com/w/cpp/container/vector/begin

As for `cbegin`, this is all inside `detail` and we wouldn't need it. Hence, I don't feel it's worth the effort, because it's internal.

Reply:
OK